packages/server/src/threads/automation.js

```diff
@@ -107,25 +107,28 @@ class Orchestrator {
     let loopSteps = []
     for (let step of automation.definition.steps) {
       stepCount++
-      let input
       if (step.stepId === LOOP_STEP_ID) {
         loopStep = step
         loopStepNumber = stepCount
         continue
       }
 
       if (loopStep) {
-        input = await processObject(loopStep.inputs, this._context)
+        // lets first of all handle the input
+        if (
+          typeof loopStep.inputs.binding === "string" &&
+          loopStep.inputs.option === "String"
+        ) {
+          loopStep.inputs.binding = loopStep.inputs.binding.split("\n")
```

Comment:
Talked about this offline - think this should handle CSV data as well as newline separated, might also be worth adding a tooltip to the input for bindings/values to tell the user the format it will split on.